src/main/external-resources/renderers/Samsung-OLED.conf

```diff
@@ -1,64 +0,0 @@
-#----------------------------------------------------------------------------
```

Comment:
Can you restore this before merge? You can use `LoadingPriority` to make yours load first

Reply:
Just when I will return from holiday abroad. 😎

Reply:
Cool where are you visiting?

Reply:
In Bulgaria